File: src/basics/Weather.java
package basics;

import java.awt.*;

import static java.awt.Color.*;

public class Weather {

    public void updateWeather(int degrees) {
        String description;
        Color color;
        if(degrees < 5) {
            description = "cold";
            color = BLUE;
        } else if (degrees < 23) {
            description = "mild";
            color = ORANGE;
        } else {
            description = "hot";
            color = RED;
        }
        // ...
    }
}

File: src/basics/Weather.kt
package basics

import java.awt.*

import java.awt.Color.*

class Weather {

    fun updateWeather(degrees: Int) {
        val (description, color) = when {
            degrees < 5 -> "cold" to BLUE
            degrees < 23 -> "mild" to ORANGE
            else -> "hot" to RED
        }
        // ...
    }
}